wagtail/contrib/modeladmin/helpers/url.py

```diff
@@ -6,19 +6,24 @@
 
 
 class AdminURLHelper:
-    def __init__(self, model):
+    def __init__(self, model, base_url_path=None):
         self.model = model
         self.opts = model._meta
+        self.base_url_path = self._get_base_url_path(base_url_path)
+
+    def _get_base_url_path(self, base_url):
```

Comment:
for consistency, I think this arg should have the same name
```suggestion
    def _get_base_url_path(self, base_url_path):
```